vignettes/demo.Rmd

```diff
@@ -315,33 +340,19 @@ cat(news, sep = "\n")
 
 Some of the intermediate commit messages are not relevant in the release notes for this release.
 We need to edit `NEWS.md` to convert the changelog to meaningful release notes.
+E.g. in real life we might [re-organize bullets](https://style.tidyverse.org/news.html#organisation-3).
 
-```{r patch-news-edit}
-length(news) <- 5
-news[3:5] <- c(
```

Comment:
I found this a bit complicated.

Reply:
Yeah. The release process changes with #27, which we'll merge right after CRAN release. No need to spend too much time on this part of the vignette.